koa_crud/src/helpers/pagination/index.ts

```diff
@@ -30,19 +30,20 @@ export const paginate = async (
   limit: number | null,
   after: string | null,
   fieldOptions: any,
+  filters: _FilterQuery<any> | null,
 ): Promise<Connection<Record<string, any>>> => {
   const cursor = after ? fromCursorHash(after) : null;
 
   const result = cursor
     ? await model
-        .find({}, { ...fieldOptions })
+        .find({ ...filters }, { ...fieldOptions })
```

Comment:
@jmrcmendoza  just put `filters` here since you don't need mutations after using spread operator